src/json.js

```diff
@@ -134,6 +134,27 @@ export function tryParseJson(json, opt_onFailed) {
   }
 }
 
+/**
+ * Helper method to get the json config from an element <script> tag
+ * Throws error if there is no or more than one <script> tag,
+ * or <script> tag has type value not equal to application/json
+ * or cannot parse the json object
+ * @param {!Element} element
+ * @return {?JsonObject}
+ */
+export function getSingleChildJsonConfig(element) {
+  const scripts = childElementsByTag(element, 'script');
+  if (scripts.length != 1) {
+    throw new Error('should have (only) one <script> child');
+  }
+  const script = scripts[0];
+  if (!isJsonScriptTag(script)) {
+    throw new Error('config should be put in a <script> tag with type = ' +
+        '"application/json"');
+  }
+  return parseJson(script.textContent);
```

Comment:
What error message does this result in? Consider using `tryParseJson` and throwing with a custom error message instead, e.g. "Failed to parse <script> contents. Is it valid JSON?"

Reply:
Good point!
Note that I didn't use `tryParseJson`, but used
```
try {
  parseJson
} catch ...
```
to avoid a type check error. Because I don't want to return undefined.